src/Core_Fix_ResourceUnloadOptimizations/Core.ResourceUnloadOptimizations.cs

```diff
@@ -28,13 +28,21 @@ public partial class ResourceUnloadOptimizations
         public static ConfigEntry<bool> OptimizeMemoryUsage { get; private set; }
         public static ConfigEntry<int> PercentMemoryThreshold { get; private set; }
         public static ConfigEntry<int> PercentMemoryThresholdDuringLoad { get; private set; }
-        
+
+#if !SBPR
+        private const int DefaultPercentMemoryThresholdDuringLoad = 65;
+        private const string PercentMemoryThresholdDuringLoadRelative = "lower";
+#else
+        private const int DefaultPercentMemoryThresholdDuringLoad = 80;
+        private const string PercentMemoryThresholdDuringLoadRelative = "higher";
+#endif
+
         internal void Awake()
         {
             DisableUnload = Config.Bind(Utilities.ConfigSectionTweaks, "Disable Resource Unload", false, new ConfigDescription("Disables all resource unloading. Requires large amounts of RAM or will likely crash your game.", null, new ConfigurationManagerAttributes { IsAdvanced = true }));
             OptimizeMemoryUsage = Config.Bind(Utilities.ConfigSectionTweaks, "Optimize Memory Usage", true, new ConfigDescription("Use more memory (if available) in order to load the game faster and reduce random stutter."));
             PercentMemoryThreshold = Config.Bind(Utilities.ConfigSectionTweaks, "Optimize Memory Threshold", 75, new ConfigDescription("Minimum amount of memory to be used before resource unloading will run.", null, new ConfigurationManagerAttributes {IsAdvanced = true}));
-            PercentMemoryThresholdDuringLoad = Config.Bind(Utilities.ConfigSectionTweaks, "Optimize Memory Threshold During Load", 65, new ConfigDescription("Minimum amount of memory to be used during load before resource unloading will run (should be lower than 'Percent Memory Threshold').", null, new ConfigurationManagerAttributes {IsAdvanced = true}));
+            PercentMemoryThresholdDuringLoad = Config.Bind(Utilities.ConfigSectionTweaks, "Optimize Memory Threshold During Load", DefaultPercentMemoryThresholdDuringLoad, new ConfigDescription($"Minimum amount of memory to be used during load before resource unloading will run (should be {PercentMemoryThresholdDuringLoadRelative} than 'Optimize Memory Threshold').", null, new ConfigurationManagerAttributes {IsAdvanced = true}));
```

Comment:
It might be cleaner to if/else this line instead of creating constants.

Reply:
I thought this was cleaner, but I can change if you want

Reply:
Yeah, I think change it, the issue I see is that this setting ends up working in two different ways which can become confusing. Ideally the setting would work the same but it is what it is.